### PR TITLE
Updated folder structure for API

### PR DESCRIPTION
**Pull request by manas-NTech**

### CODE REVIEW

File: go1percent/test/Go1PercentAPIAutomation/myProfile/KnoldusLogo.png
Comment (Ritika-gupta20) [inline go1percent/test/Go1PercentAPIAutomation/myProfile/KnoldusLogo.png:1]: Please put this file in helpers folder under Go1PercentAPIAutomation->yourfolder->file.png

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Please move the image file to helpers under API/your module

File: go1percent/test/Go1PercentAPIAutomation/myProfile/KnoldusLogo.png
Comment (manas-NTech) [reply go1percent/test/Go1PercentAPIAutomation/myProfile/KnoldusLogo.png:1]: Resolved.

File: go1percent/KnoldusLogo.png
Comment (Ritika-gupta20) [inline go1percent/KnoldusLogo.png:1]: Please put this file under the folder helpers->yourfolder->image.png

**Review by Ritika-gupta20 — CHANGES_REQUESTED** [review]
Please move image to correct folder

File: go1percent/KnoldusLogo.png
Comment (manas-NTech) [reply go1percent/KnoldusLogo.png:1]: Updated.

**Review by deepansh257 — APPROVED** [review]

**Review by Sparsh79 — APPROVED** [review]

**Review by Ritika-gupta20 — APPROVED** [review]

**Pull request merged**